sklearn/naive_bayes.py

```diff
@@ -822,7 +834,13 @@ class ComplementNB(_BaseDiscreteNB):
     Parameters
     ----------
     alpha : float, default=1.0
-        Additive (Laplace/Lidstone) smoothing parameter (0 for no smoothing).
+        Additive (Laplace/Lidstone) smoothing parameter
+        (set alpha=0 and force_alpha=True, for no smoothing).
+
+    force_alpha : bool, default=False
+        If false and alpha is too close to 0, it will set alpha to _ALPHA_MIN.
+        If true, warn user about potential numeric errors
+        and proceed with alpha unchanged.
```

Comment:
Just a small correction.

```suggestion
    force_alpha : bool, default=False
        If False and alpha is too close to 0, it will set alpha to `_ALPHA_MIN`.
        If True, warn user about potential numeric errors
        and proceed with alpha unchanged.

        .. versionadded:: 0.24
```